Automattic/wp-calypso
# Upgrade webpack-rtl-plugin to 1.8.0 which upgrades cssnano from v3 to v4

Allows us to revert changes from #29288, as cssnano v4 no longer does unsafe optimizations
by default. In v3, we had to disable them manually. With v4 we can go back to default config.

See also romainberger/webpack-rtl-plugin#26 which is the cssnano upgrade PR that was shipped in v1.8.0.

Good news for @dmsnell: the cssnano upgrade removed a lot of PostCSS package duplication and my `node_modules` went down from 473MB to 418MB. That's 12%.

**How to test:**
Verify that no CSS unsafe optimizations are done. One of the best ways is to check if the Gutenberg Publish button animation bug in #29267 didn't regress.
